process/smartContract/process.go

```diff
@@ -199,9 +201,12 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return nil
 	}
 
-	resolved, err := sc.resolveBuiltInFunctions(tx, acntSnd, acntDst, vmInput)
-	if err != nil || resolved {
-		log.Debug("processed build in functions")
+	executed, err := sc.resolveBuiltInFunctions(tx, acntSnd, acntDst, vmInput)
+	if err != nil {
+		log.Debug("processed built in functions error", "error", err.Error())
```

Comment:
You can change from log.Trace to log.Debug the error from line 227 (processVMOutput) as it remains the only one error from this method logged with Trace

Reply:
that is the most common error, it appears for any bad sc transaction, and when using tx generator it makes a lot of noise in the log. This is why that is trace.

Reply:
ok